File: app/src/main/java/com/babestudios/companieshouse/data/model/company/PreviousCompanyName.java
package com.babestudios.companieshouse.data.model.company;

import com.google.gson.annotations.SerializedName;

public class PreviousCompanyName {

	@SerializedName("ceased_on")
	public String ceasedOn;

	@SerializedName("effective_from")
	public String effectiveFrom;

	@SerializedName("name")
	public String name;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/PreviousCompanyName.kt
package com.babestudios.companyinfouk.data.model.company

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class PreviousCompanyName(

		@SerializedName("ceased_on")
		var ceasedOn: String? = null,

		@SerializedName("effective_from")
		var effectiveFrom: String? = null,

		@SerializedName("name")
		var name: String? = null

) : Parcelable